x/metadata/legacy/v042/store.go

```diff
@@ -7,23 +7,29 @@ import (
 )
 
 func MigrateAddresses(ctx sdk.Context, storeKey sdk.StoreKey) error {
+	ctx.Logger().Info("Migrating Metadata OSLocators (1/5)")
 	err := MigrateOSLocatorKeys(ctx, storeKey)
 	if err != nil {
 		return err
 	}
+	ctx.Logger().Info("Migrating Metadata Scopes (2/5)")
 	err = MigrateAddressScopeCacheKey(ctx, storeKey)
 	if err != nil {
 		return err
 	}
+	ctx.Logger().Info("Migrating Metadata Value Owner Scopes (3/5)")
 	err = MigrateValueOwnerScopeCacheKey(ctx, storeKey)
 	if err != nil {
 		return err
 	}
+	ctx.Logger().Info("Migrating Metadata Scope Specs (4/5)")
 	err = MigrateAddressScopeSpecCacheKey(ctx, storeKey)
 	if err != nil {
 		return err
 	}
-	return MigrateAddressContractSpecCacheKey(ctx, storeKey)
+	ctx.Logger().Info("Migrating Metadata Contract Specs (5/5)")
+	err = MigrateAddressContractSpecCacheKey(ctx, storeKey)
```

Comment:
#nit this could just be:
```golang
return MigrateAddressContractSpecCacheKey(ctx, storeKey)
```

Reply:
Oh, yes.  This was overlooked.  I think I was going to put a logging statement after that but removed it.